integr8ly/application-monitoring-operator
# wait for operator before deploying crs

Wait for the prometheus operator to be ready (and finish deployment of it's CRDs) before continuing to install Prometheus and Alertmanager.

Fixes https://github.com/integr8ly/application-monitoring-operator/issues/24

Verification steps:
1. Use a fresh cluster where the images have not been pulled yet
1. Use a namespace called `application-monitoring` or change the namespace here: https://github.com/integr8ly/application-monitoring-operator/blob/master/deploy/roles/prometheus-operator-clusterrole_binding.yaml#L12
1. Run the operator locally (or deploy it) against this namespace and create the ApplicationMonitoring CR: `oc create -f deploy/examples/ApplicationMonitoring.yaml`
1. None of the Operators should fail and the stack should be deployed
